product/scripts/tmcmp.sh

```diff
@@ -20,9 +20,9 @@ function mvn_test {
     printf "%4s %8s %8s %10d %10d %7d %7d %6d %5b %8s %5s %8d %8b %8d\n" \
       "host" "04:10:34" $product -1 $iteration $warmUp $thread -1 $jts $xstore $aio $bufferSize $syncDelete $flushRate
   else
-    echo "TEST: $testrun of $testcount: mvn test -P $product -Diterations=$iteration -Dthreads=$thread -Djts=$jts -DwarmUpIterations=$warmUp -DobjectStoreDir=$storeDir $@"
+    echo "TEST: $testrun of $testcount: ./build.sh test -P $product -Diterations=$iteration -Dthreads=$thread -Djts=$jts -DwarmUpIterations=$warmUp -DobjectStoreDir=$storeDir $@"
```

Comment:
This means that the script can only be called from the top level directory. That seems ok but I cannot find anything that calls the script in the repo so assume that it is used by the jobs although I did not check.